incubator/virtualcluster/pkg/util/cluster/cluster.go

```diff
@@ -173,7 +173,7 @@ func (c *Cluster) getMapper() (meta.RESTMapper, error) {
 		return c.mapper, nil
 	}
 
-	mapper, err := apiutil.NewDiscoveryRESTMapper(c.RestConfig)
+	mapper, err := apiutil.NewDynamicRESTMapper(c.RestConfig)
```

Comment:
Can you please explain why this needs to be changed?

Reply:
@Fei-Guo, Without this change, if CRD got created after virtual cluster (VC) is initialized ---- VC cache and informer has been constructed with standard resource types, New CRD (informer) type is not mapped into VC cache. The consequence is that, even thought CRD got synchronized from super to VC properly, but CR informer cannot work with un-updated cache, and CR synchronization does not work. 
By using the dynamic rest mapper, anytime a new CRD is created in VC, the VC cache will dynamically map the new CRD type to its cache ,  CR informer (used by CR syncer) can link with updated cache and work properly.